Sources/StripeKit/Core Resources/Balance/BalanceRoutes.swift

```diff
@@ -35,7 +24,9 @@ public struct StripeBalanceRoutes: BalanceRoutes {
         self.apiHandler = apiHandler
     }
     
-    public func retrieve() -> EventLoopFuture<StripeBalance> {
-        return apiHandler.send(method: .GET, path: balance, headers: headers)
+    public func retrieve() async throws -> Balance {
+        try await apiHandler.send(method: .GET,
+                                  path: balance,
+                                  headers: headers)
```

Comment:
Are you deliberately removing ELF support from the package?

Reply:
Correct. I'm also considering letting this be the main package with conditional vapor support. Something like:
```swift
#IfCanImport(Vapor)
// vapor specific support like webhook signature verification and Request and Application helpers.
#endIf
```